test/govuk_component/task_list_test.rb

```diff
@@ -14,6 +14,20 @@ def simple_tasklist
             {
               type: 'paragraph',
               text: 'This is the first paragraph'
+            },
+            {
+              type: 'list',
+              links: [
+                {
+                  path: '/link1',
+                  text: 'Link 1',
+                },
+                {
+                  path: '/link2',
+                  text: 'Link 2',
+                  cost: '&pound;0 to &pound;300'
```

Comment:
Talked about in person: Could cost be 'context' or 'info' something that allows this to be generic?